dspace-api/src/main/java/org/dspace/app/audit/AuditConsumer.java

```diff
@@ -0,0 +1,75 @@
+/**
+ * The contents of this file are subject to the license and copyright
+ * detailed in the LICENSE and NOTICE files at the root of the source
+ * tree and available online at
+ *
+ * http://www.dspace.org/license/
+ */
+package org.dspace.app.audit;
+
+import java.util.List;
+import java.util.UUID;
+
+import org.dspace.core.Context;
+import org.dspace.event.Consumer;
+import org.dspace.event.Event;
+import org.dspace.services.ConfigurationService;
+import org.dspace.utils.DSpace;
+
+/**
+ * Class to store all received events in an audit log
+ * 
+ * @version $Revision$
```

Comment:
`$Revision$` has no meaning in Git, and its CVS/SVN value is not what `@version` means.  We should not use `$Revision$` in new code and should eventually remove it from old code.